eradiate/experiments/_core.py

```diff
@@ -198,6 +198,10 @@ class Experiment(ABC):
         type="dict[str, dataset]",
     )
 
+    @property
+    def integrator(self) -> Integrator:
```

Comment:
Let's add a docstring here:
```suggestion
    def integrator(self) -> Integrator:
        """
        :class:`.Integrator`: Integrator used to solve the radiative transfer equation.
        """
```